ogre2/src/Ogre2SelectionBuffer.cc

```diff
@@ -222,18 +222,24 @@ void Ogre2SelectionBuffer::CreateRTTBuffer()
 
   Ogre::TextureGpuManager *textureMgr =
     ogreRoot->getRenderSystem()->getTextureGpuManager();
+  std::string selectionTextureName = "SelectionPassTex";
```

Comment:
@iche033 nit: can be made `const`

Reply:
made `const` in [4416657](https://github.com/gazebosim/gz-rendering/pull/666/commits/44166572bd25929d9bb526dfa280721b9c935d0a)